### PR TITLE
Create integer_to_roman.md

### LINKED ISSUE #8000: Create integer_to_roman.md
### Description

I'll create a file to convert integer to roman numerals

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Preet1406**
<hr>

## Description 📜

I've added a file to convert integer to roman numeral. 
Fixes #8000  

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Preet1406** [comment]
@apeksha235  please review it

**Comment by Preet1406** [comment]
@apeksha235 @kashishahuja2002 please review it

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@Preet1406 Add the relative link of your file in the readme of the respective folder

File: C_CPP/Number System/Conversion-Programs/integer_to_roman.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+![romanToInt](https://user-images.githubusercontent.com/73405757/140963448-360c6a7f-f391-4396-8fbd-68ab3c365c52.png)  
+Source: [LeetCode](https://leetcode.com/problems/integer-to-roman/)
+        
+## Code of converting Integer to roman numerals in C++  
+```c++
+#include <bits/stdc++.h>
+using namespace std;
+
+string intToRoman(int N) {
+    int val[13] = {1000,900,500,400,100,90,50,40,10,9,5,4,1};
```
Comment (apeksha235) [inline C_CPP/Number System/Conversion-Programs/integer_to_roman.md:18]: add relevant comments to your code

File: C_CPP/Number System/Conversion-Programs/integer_to_roman.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    cout << "Enter Number: ";
+    cin >> number;
+    cout << "Roman Numeral is: " << intToRoman(number);
+    return 0;
+}
+```  
+## Input/Output Examples
+
+#### Example 1:
+Input: number = 3
```
Comment (apeksha235) [inline C_CPP/Number System/Conversion-Programs/integer_to_roman.md:43]: add the input and output inside the highlighter ```

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@Preet1406 make these changes

**Comment by Preet1406** [comment]
@apeksha235 Changes done, for the link, there is no readme file in the respective folder. Please check

**Comment by apeksha235** [comment]
@Preet1406 
image

Add the link under the ```conversion programs``` as a subheading

**Comment by Preet1406** [comment]
@apeksha235 Please check again

**Review by apeksha235 — APPROVED** [review]
@Preet1406 Good!! 
@Saviour1001 please merge this

**Pull request merged**